### PR TITLE
Add pause, terminate process upon tab close, remove double play

### PR DESCRIPTION
**Pull request by demeng**
added marks to audio buffers.
Play buffers that have specific marks to reduce latency
--idea by @Dennis

Result:
small delay, no longer significant
smoother transition

### CODE REVIEW

File: background/background.js [hunk 1]
```diff
@@ -98,6 +107,11 @@ class AudioProcessor {
       sourceBuffer.sampleRate = inputBuffer.sampleRate;
       sourceBuffer.length = inputBuffer.length;
       sourceBuffer.startingUp = this.startingUp;
+      sourceBuffer.postPaused = this.postPaused;
+      sourceBuffer.tabStatus = this.tabStatus;
+      messagePostCount++;
+      logger.log(`new Message posted ${messagePostCount}`, this.tabId);
+      
       this.port.postMessage(sourceBuffer);
```
Comment (demeng) [inline background/background.js]: Error returned from this line while running on Mac: "Uncaught Error: Attempting to use a disconnected port object at ScriptProcessorNode.scriptNode.onaudioprocess (background.js:115)". @CJtheLin

File: host/chromeaudio.py [hunk 2]
```diff
@@ -4,15 +4,20 @@
 import json
 import struct
 
+path = '/home/kernellabs3/Desktop/python_output'
```
Comment (demeng) [inline host/chromeaudio.py]: These two lines 7&8 can be deleted. @CJtheLin

Comment (CJtheLin) [reply host/chromeaudio.py]: deleted

File: background/background.js (hunk 1 above)
Comment (CJtheLin) [reply background/background.js]: disconnected port is caused by the error in python line 7&8. Problem fixed, now it can work properly on mac terminal

**Pull request merged**